.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Dataflow_Java11.groovy

```diff
@@ -20,26 +20,40 @@ import CommonJobProperties as commonJobProperties
 import PostcommitJobBuilder
 
 
-PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java11_ValidatesRunner_Dataflow',
+PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java_ValidatesRunner_Dataflow_Java11',
   'Run Dataflow ValidatesRunner Java 11', 'Google Cloud Dataflow Runner ValidatesRunner Tests On Java 11', this) {
 
   description('Runs the ValidatesRunner suite on the Dataflow runner with Java 11 worker harness.')
 
-  commonJobProperties.setTopLevelMainJobProperties(delegate, 'master', 270)
+  def JAVA_11_HOME = '/usr/lib/jvm/java-11-openjdk-amd64'
```

Comment:
Can we preferably use a Jenkins environment variable for the [Java home locations](https://issues.apache.org/jira/browse/BEAM-8919) here and in the [direct runner Java 11 case](https://github.com/apache/beam/blob/19dcb5c10c270b367cb4f897090eae2f700b11a6/.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Direct_Java11.groovy#L30)?

Reply:
I'm not sure if such variables exist, but I agree it would be better to use those instead of the paths. Maybe after Jenkins for Beam is up and running we'll be able to add the envs